### ISSUE TITLE
[Accessibility]: Icons are undetectable by screenreader

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5530]
### Audit type

Screen reader navigation

### User journey audit issue

#5500

### Problem description

Currently, screenreaders either skip over or read images on the site as "image".

### Expected behavior

All imagery should be detectable for a11y. Our icons are a part of our rich content. They should include an aria label or alt tag to be detectable to screenreaders.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
